packages/ast/src/encoding/proto/types.ts

```diff
@@ -170,6 +170,10 @@ export const getFieldOptionality = (
 ) => {
     const isScalarDefaultToNullable = context.pluginValue('prototypes.isScalarDefaultToNullable');
 
+    if(field?.options?.['(gogoproto.moretags)'] && getOptionDeprecated(field?.options?.['(gogoproto.moretags)']) === 'false'){
```

Comment:
correct logic here should be:

    if(field?.options?.['(gogoproto.moretags)'] && getOptionDeprecated(field?.options?.['(gogoproto.moretags)']) === 'true'){
        return true
    }